Pod/Pod/ViewController.m

```diff
@@ -0,0 +1,66 @@
+#import "ViewController.h"
+#import "HYPMediaPageViewController.h"
+
+static const CGFloat HYPMediaPageViewHeight = 357.0f;
+
+@interface ViewController ()
+
+@property (nonatomic, strong) HYPMediaPageViewController *mediaPageViewController;
+
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+
+    self.view.backgroundColor = [UIColor blackColor];
+
+    [self.view addSubview:self.mediaPageViewController.view];
+
+    NSDictionary *views = @{@"mediaPageView": self.mediaPageViewController.view};
+
+    [self.view addConstraints:
+     [NSLayoutConstraint constraintsWithVisualFormat:@"H:|[mediaPageView]|"
+                                             options:0
+                                             metrics:nil
+                                               views:views]];
+
+    [self.view addConstraint:
+     [NSLayoutConstraint constraintWithItem:self.mediaPageViewController.view
+                                  attribute:NSLayoutAttributeCenterY
+                                  relatedBy:NSLayoutRelationEqual
+                                     toItem:self.view
+                                  attribute:NSLayoutAttributeCenterY
+                                 multiplier:1.0f
+                                   constant:0.0f]];
+}
+
+#pragma mark - Getters
+
+- (HYPMediaPageViewController *)mediaPageViewController
+{
+    if (_mediaPageViewController) return _mediaPageViewController;
+
+    _mediaPageViewController = [HYPMediaPageViewController new];
+    _mediaPageViewController.view.translatesAutoresizingMaskIntoConstraints = NO;
```

Comment:
It does not have any special initializer, this is just the demo use. Or perhaps I misunderstood you?